engine/src/main/java/org/terasology/engine/network/internal/pipelineFactory/InfoRequestPipelineFactory.java

```diff
@@ -22,7 +24,7 @@
  * A pipeline that requests {@link org.terasology.engine.network.ServerInfoMessage} before it auto-disconnects. This is similar
  * to {@link TerasologyClientPipelineFactory}.
  */
-public class InfoRequestPipelineFactory extends ChannelInitializer {
+public class  InfoRequestPipelineFactory extends ChannelInitializer {
```

Comment:
Please double check for checkstyle alerts and formatting (I can spot an additional space here :yum: )